app/views/support/LandingPageOps.scala

```diff
@@ -35,6 +39,24 @@ object LandingPageOps {
     def getIncentiveTermsAndConditions: String = {
       promotion.asIncentive.fold("") { p => p.promotionType.termsAndConditions.mkString }
     }
+    def getIncentiveLegalTerms: String = {
+      promotion.asIncentive.fold("") { p => p.promotionType.legalTerms.mkString }
```

Comment:
would it be clearer:
`promotion.asIncentive.map(_.promotionType.legalTerms.mkString).getOrElse("")`